packages/app/src/systems/Settings/pages/ViewRecoveryPhrase/ViewRecoveryPhrase.tsx

```diff
@@ -0,0 +1,72 @@
+import { cssObj } from '@fuel-ui/css';
+import { Box, Button, Flex, Dialog, Text, Alert } from '@fuel-ui/react';
+
+import { useExportVault } from '../../hooks';
+
+import { Mnemonic } from '~/systems/Core';
+import { useOverlay } from '~/systems/Overlay';
+import { UnlockCard } from '~/systems/Unlock';
+
+export function ViewRecoveryPhrase() {
+  const { close } = useOverlay();
+  const { handlers, error, isUnlockOpened, isLoading, words } =
+    useExportVault();
+
+  if (isUnlockOpened) {
+    return (
+      <UnlockCard
+        onUnlock={(password) => {
+          handlers.exportVault(password);
+        }}
+        headerText="Confirm your Password"
+        onClose={() => close()}
+        unlockError={error}
+        isLoading={isLoading}
+      />
+    );
+  }
+
+  return (
+    <>
+      <Dialog.Heading>Recovery Phrase</Dialog.Heading>
+      <Dialog.Description as="div">
+        <Flex gap="$4" direction="column" align="center">
+          <Box css={styles.mnemonicWrapper}>
+            <Mnemonic type="read" value={words} />
+          </Box>
+          <Alert status="warning" css={styles.alert}>
+            <Alert.Description>
+              <Text fontSize="xs" css={styles.alertDescription}>
+                Do not share your Recovery Phrase. {'\n'}
+                The recovery phrase, provide access to all your accounts sharing
+                or losing it may result in permanent loss of funds.
+              </Text>
```

Comment:
Change text to:
```
DON'T SHARE your Recovery Phrase.

This phrase provides access to all of you accounts. Sharing or losing it may result in permanent loss of funds.
```